models/Category.php

```diff
@@ -2,7 +2,7 @@
 
 use Str;
 use Model;
-use URL;
+use Url;
```

Comment:
Why was this changed?

Reply:
Because of https://github.com/rainlab/blog-plugin/blob/master/models/Post.php#L4